spring-cloud-dataflow-audit/src/test/java/org/springframework/cloud/dataflow/server/audit/service/DefaultAuditRecordServiceTests.java

```diff
@@ -33,6 +35,8 @@
 import org.springframework.data.domain.PageRequest;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
```

Comment:
Wdyt about yanking this old Junit4 out in favor of Junit5 and AssertJ assertions?

Reply:
I am ok doing that in a later PR as well.

Reply:
Will revisit the tests and upgrade to JUnit 5 at some point.

Reply:
In the meantime let's not pull in `org.junit.*` classes and instead use AssertJ or Jupiter APIs.